ansible/roles/sunbird-system-init/templates/system_init_commands.j2

```diff
@@ -0,0 +1,336 @@
+#!/bin/bash
+sunbird_custodian_tenant_name="{{sunbird_custodian_tenant_name}}"
+sunbird_custodian_tenant_description="{{sunbird_custodian_tenant_description}}"
+sunbird_custodian_tenant_channel="{{sunbird_custodian_tenant_channel}}"
+sunbird_root_user_firstname="{{sunbird_root_user_firstname}}"
+sunbird_root_user_lastname="{{sunbird_root_user_lastname}}"
+sunbird_root_user_username="{{sunbird_root_user_username}}"
+sunbird_root_user_password="{{sunbird_root_user_password}}"
+sunbird_root_user_email="{{sunbird_root_user_email}}"
+sunbird_root_user_phone="{{sunbird_root_user_phone}}"
+ekstep_proxy_base_url="{{ekstep_proxy_base_url}}"
+ekstep_api_key="{{ekstep_api_key}}"
+sunbird_es_host="{{sunbird_es_host}}"
+proto="{{proto}}"
```

Comment:
Remove proto variable and use the "http" or "https" in the respective URL.

Reply:
That will be problematic, as we don't know which protocol is the user is opting for. Because Sunbird supports both HTTP and https installation.